### PR TITLE
Sidetrack sounds

### PR DESCRIPTION
**Pull request by DanielF-Endless**
Adding sounds to the sidetrack toy app.
T26415 - https://phabricator.endlessm.com/T26415

### CODE REVIEW

File: data/metadata.json [hunk 1]
```diff
@@ -1156,6 +1156,77 @@
...
+    },
+    "sidetrack/sfx/robot": {
+        "sound-file": "sidetrack/sfx/robot.wav"
+    },
+    "sidetrack/sfx/pit_fall": {
+        "sound-file": "sidetrack/sfx/failure.wav"
+    },
+    "sidetrack/sfx/wall_bump": {
+        "sound-file": "sidetrack/sfx/failure.wav"
+    }
```
Comment (cfoch) [inline data/metadata.json]: Add `,` here :-)

Comment (DanielF-Endless) [reply data/metadata.json]: Oh jeez, thx Fabian! Changes made.

File: data/metadata.json [hunk 2]
```diff
@@ -1156,6 +1156,77 @@
     "quests/transfer-complete": {
         "sound-file": "quests/transfer-complete.webm"
     },
+    "sidetrack/bg/manual_mode": {
+        "sound-file": "sidetrack/bg/bg1.wav",
```
Comment (cfoch) [inline data/metadata.json]: Please convert wav files to webm.
Use `./tools/convert-sounds` script.

**Comment by cfoch** [comment]
Also sort the metadata file using `./tools/sort-metadata.py --inplace`

File: data/metadata.json (hunk 2 above)
Comment (DanielF-Endless) [reply data/metadata.json]: wav files converted in latest commit.

**Pull request merged**